webpack.config.js

```diff
@@ -36,7 +44,10 @@ module.exports = {
         jQuery: "jquery"
       })
    ],
-   entry: './public/js/main.js',
+   entry: [
+     './public/js/main.js',
+     './views/standard/less/main.less'
```

Comment:
So my original plan was to add an
```js
import './views/standard/less/main.less';
```
to the list of other imports we have in `public/js/main.js` and then have less/css/style-loader work their magic so that when the bundle.js file is loaded, those styles would be placed in a `<style>` tag at the head of the document.

But then I read you could just list the less file as an entry file to the bundle and everything would work just the same without having to import a less file in our JS.

It's kind of neat but I'm not sure if it's too magicky. On the other hand though, there isn't a single place in our application code where our style is imported with this approach, so I thought sounded nice too.

We can definitely play around with it. Maybe we could import `pull.less` in `public/js/Pull.js` such that each JS component is responsible for importing styles it expects to be available.

Reply:
Oh, I mis-interpreted the entries array here. Multiple entries means it combines them, not that it creates two separate bundles.
 Cool beans.